package.json

```diff
@@ -104,7 +105,9 @@
     "typescript": "^4.5.3"
   },
   "dependencies": {
-    "commander": "^10.0.0"
+    "commander": "^10.0.0",
+    "isomorphic-dompurify": "^1.6.0",
+    "marked": "^5.0.4"
```

Comment:
I chose Marked because it is the most popular regularly maintained Markdown to HTML converter NPM package I could find.